src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1646,29 +1697,71 @@ void PgScrubber::handle_scrub_reserve_request(OpRequestRef op)
     advance_token();
     granted = true;
 
```

Comment:
Unneeded blank line. Nothing new but, as we're already touching the area, seems worth tackling.